renderer/lib/cast.js

```diff
@@ -21,50 +22,102 @@ module.exports = {
 // Callback to notify module users when state has changed
 var update
 
-function init (callback) {
-  update = callback
-
-  // Start polling Chromecast or Airplay, whenever we're connected
-  setInterval(() => pollCastStatus(state), 1000)
-
-  // Listen for devices: Chromecast and Airplay
-  chromecasts.on('update', function (player) {
-    state.devices.chromecast = player
-    addChromecastEvents()
-  })
+// chromecast player implementation
+function chromecastPlayer (player) {
```

Comment:
looking closely my own code there is no need (as far as i see) to expose the device's internal player in the returning function, just keep the invocation methods.